dotnet/src/webdriver/BiDi/WebDriver.Extensions.cs

```diff
@@ -0,0 +1,27 @@
+using OpenQA.Selenium.BiDi.Modules.BrowsingContext;
+using System.Threading.Tasks;
+
+namespace OpenQA.Selenium.BiDi;
+
+public static class WebDriverExtensions
+{
+    public static async Task<BiDi> AsBidirectionalAsync(this IWebDriver webDriver)
```

Comment:
It shouldn't be extension, it should be a nested part of WebDriver (`IWebDriver`, `RemoteWebDriver`, ...).
Also, disposing of `BiDi` object should not close a browser.

Reply:
It is fine to keep it as is via method extensions, preventing a breaking change. Later on we can announce the functionality via extending existing interface or introducing new one.

So, here we should not close browser in `Dispose`